src/Umbraco.Forms.Integrations.Crm.ActiveCampaign/ActiveCampaignContactsWorkflow.cs

```diff
@@ -67,6 +67,13 @@ public override WorkflowExecutionStatus Execute(WorkflowExecutionContext context
                 // Check if contact exists.
                 var contacts = _contactService.Get(email).ConfigureAwait(false).GetAwaiter().GetResult();
 
+                if(contacts.Contacts.Count > 0 && !_settings.AllowContactUpdate)
+                {
+                    _logger.LogError("ActiveCampaign contact update is not allowed.");
+
+                    return WorkflowExecutionStatus.Cancelled;
```

Comment:
```suggestion
                    return WorkflowExecutionStatus.Success;
```

Workflow has still succeeded if we've not updated due to configuration settings.